resources/views/torrent/create.blade.php

```diff
@@ -91,6 +91,19 @@ class="upload-form-file form__file"
                         class="upload-form-file form__file"
                         type="file"
                         accept=".nfo"
+                    <!-- Porn category extra IDs -->
```

Comment:
These shouldn't be here. They should be moved below with rest of meta inputs and use alpine logic like others to only show if category selected is porn_meta